src/CodeGen_Hexagon.cpp

```diff
@@ -293,6 +295,9 @@ void CodeGen_Hexagon::init_module() {
         { IPICK(is_128B, Intrinsic::hexagon_V6_vsathub),  u8v1,  "trunc_satub.vh",  {i16v2} },
         { IPICK(is_128B, Intrinsic::hexagon_V6_vsatwh),   i16v1, "trunc_sath.vw",   {i32v2} },
 
+        // v62 - Saturating narrowing cast
+        { TGTPICK(is_v62, Intrinsic::hexagon_V6_vsatuwuh, Intrinsic::hexagon_V6_vsatwh),   u16v1, "trunc_satuh.vuw",   {u32v2} },
```

Comment:
This might break v60 - A large unsigned number will be treated as negative by hexagon_V6_vsatwh and will be saturated to 0 instead of UINT16_MAX.